kits/expense-budget-copilot/prompts/analyze-expenses_llmnode-233_user_1.md

```
@@ -0,0 +1,2 @@
+Transaction data: {{triggerNode_1.output.transactionText}}
+Currency:{{triggerNode_1.output.currency}}
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)